src/frontend/components/StudentLiveAdvertising/StudentLiveRegistration/RegistrationForm/index.spec.tsx

```diff
@@ -133,15 +84,11 @@ describe('<RegistrationForm />', () => {
       video: 'id',
     });
     mockCreateLiveRegistration.mockResolvedValue(liveRegistration);
-    const values = {
-      email: 'some.email@openfun.com',
-    };
 
     render(
       wrapInIntlProvider(
         <RegistrationForm
-          values={values}
-          setValues={setValues}
+          initialEmail="some.email@openfun.com"
```

Comment:
maybe we can just call it email as it can't be changed if initialized, wdyt

Reply:
Idk, with a props named email, I would expect the value to be updated if I change the prop but that is not the case here.
I can rename it anyway as it pleases you.

Reply:
it confused me when I read the code, I would go for default or just email, "initial" gives me the understanding that it will necessarily change which is not the case as it won't.